lib/gruf/newrelic/server_interceptor.rb

```diff
@@ -44,7 +44,11 @@ def call
 
       def accept_distributed_tracing
         payload = request.active_call.metadata[NEWRELIC_TRACE_HEADER]
-        ::NewRelic::Agent::DistributedTracing.accept_distributed_trace_payload(payload)
+        if payload.instance_of? Array
```

Comment:
Is there a reason you're wanting to ensure this is _exactly_ an array, as opposed to checking to see if it's `Enumerable`? This might be better as: 

```
def accept_distributed_tracing
  payload = request.active_call.metadata[NEWRELIC_TRACE_HEADER]
  payload = payload.first if payload.is_a?(Enumerable)
  ::NewRelic::Agent::DistributedTracing.accept_distributed_trace_payload(payload)
end
```